Src/FluentAssertions/Specialized/ExceptionAssertions.cs

```diff
@@ -199,7 +199,7 @@ public ExceptionAssertions(IEnumerable<TException> exceptions)
         return this;
     }
 
-    private Exception[] AssertInnerExceptionExactly(Type innerException, string because = "",
+    private IEnumerable<Exception> AssertInnerExceptionExactly(Type innerException, string because = "",
```

Comment:
This conflicts with [CA1859](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1859)